src/components/latest-scanned.tsx

```
@@ -13,44 +14,83 @@ import {
+                  <TableRow key={scan.id}>
+                    <TableCell className='max-w-[200px] font-medium truncate'>
+                      <div>{scan.student.fullname}</div>
+                    </TableCell>
+                    <TableCell>{scan.rombel.nama}</TableCell>
+                    <TableCell>
+                      {dayjs(scan.created_at)
+                        .locale("id")
+                        .format("HH:mm, DD MMMM")}
+                    </TableCell>
+                  </TableRow>
+                ))}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)